PluginsAndFeatures/azure-toolkit-for-intellij/rider/backend.gradle

```diff
@@ -2,12 +2,26 @@ import groovy.io.FileType
 
 def backendGroup = 'backend'
 
+def riderSdkPackageVersion = {
```

Comment:
👍

Reply:
Doesn't seem to resolve well [in tests](https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_AzureToolsForIntellij_RunRiderTests/3770890?showRootCauses=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&showLog=3770890_1261_620.1266&logFilter=debug)... Investigating.